### DISCUSSION TITLE
facing some issues

### DISCUSSION THREAD

**Discussion by shrutilenka** [roopeshsn/freshbey#64] [General]
![ss](https://user-images.githubusercontent.com/96660357/193803391-640eb2cf-6ccf-48ed-a52d-3d1895e93279.png)

I installed all the dependencies mentioned in the README file, but a few of the landing page elements are missing. I would be grateful if you could help me out with this @roopeshsn

**Comment by roopeshsn** [comment]
Have you created your own MongoDB cluster?

**Reply by shrutilenka** [reply]
sir, basically I want to contribute to the frontend and I have no idea about the MongoDB cluster. can you guide me through this?
